# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/editing/run/backcolor.html.ini
# Commit: 554d63dad12e
# Full Hash: 554d63dad12e4d1f2560e99de1fafac9a97da026
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2023-09-27 09:02:41
# Regressor Bug: 1851951
# File Overlap Count: 1
# Description:
#   Bug 1851951 - Make `HTMLEditor` refer computed `display` instead of the HTML default style at considering block or inline element r=m_kato
#   
#   Blink [1] and WebKit [2] refers `display-outside` value when they consider whether an
#   element is a block or an inline.
#   
# ==============================================================================

diff -r 97dc7be3b93c -r 554d63dad12e testing/web-platform/meta/editing/run/backcolor.html.ini
--- a/testing/web-platform/meta/editing/run/backcolor.html.ini	Wed Sep 27 02:50:33 2023 +0000
+++ b/testing/web-platform/meta/editing/run/backcolor.html.ini	Wed Sep 27 03:01:26 2023 +0000
@@ -158,6 +158,10 @@
   [[["stylewithcss","true"\],["backcolor","#00FFFF"\]\] "<span style=\\"display: block; background-color: aqua\\"><span style=\\"display: block; background-color: tan\\">b[ar\]</span></span>" queryCommandValue("backcolor") before]
     expected: FAIL
 
+  [[["stylewithcss","false"\],["backcolor","#00FFFF"\]\] "<span style=\\"display: block; background-color: aqua\\"><span style=\\"display: block; background-color: tan\\">b[ar\]</span></span>" queryCommandValue("backcolor") before]
+    expected:
+      if early_beta_or_earlier: FAIL
+
   [[["stylewithcss","false"\],["backcolor","#00FFFF"\]\] "<span style=\\"display: block; background-color: aqua\\"><span style=\\"display: block; background-color: tan\\">b[ar\]</span></span>" queryCommandValue("backcolor") after]
     expected: FAIL
 